virus/hespivirus_rivalense.md

```diff
@@ -0,0 +1,273 @@
+spiders, with the first found in spinybacked orbweaver spiders [13].
+
+**Ants.** The last run SRR12918481 is from a pool of five *Pheidole megacephala* (big-headed ant) collected at Molokai,
+Hawaii [4]. However, its palmprint only has 91% identity with the other runs, which otherwise agree perfectly, and an
+estimated coverage of 1, which falls short of the coverage quality recommended by [14]. Thus, we discard the SRR12918481
+hit for our analyses, both for simplicity and due to its low confidence. It could be worthwhile to revisit SRR12918481
+in future work. Brettell et al. [4] found that *P. megacephala* , along with other apiary pests, may act as viral
+reservoirs for bee-associated viruses. Given that *P. megacephala* is a highly invasive pest beyond apiaries, a similar
+inter-species transmission could be a possibility with *L. hesperus*.
+
+### Viral genome
+
+Our contig `Hriv_contig` is 10924 nt long with a 39.4% GC content. We observe a 42 nt A stretch towards the 3’ end of
+the contig, which we hypothesize to be a poly(A) tail. This is due to the recorded presence of poly(A) tails in related
+viruses, such as Negeviruses [6], and the use of Oligo-dT selection in SRR17250291. However, the stretch is followed by
+a 41 nt suffix of non-repeating bases. To check for potential assembly errors, we realign the reads from SRR17250291 to
+our contig using bowtie2 [19], resulting in a mean 1026x coverage. We observe high coverage (~300x) of the bases
+following the poly(A) stretch, which decays to 1x over the next 20 bases. However, there is a coverage bias of ends of
+contigs due to the assembly process, and a similar decay is observed in the 5’ end. Hence, our analysis remains
+inconclusive.
+
+We find 10 ORFs in our viral contig with ORFfinder and restricting to 75 nt non-nested ORFs. InterproScan [20] predicts
+domains in 3 of 10 ORFs, ignoring disorder predictions, coils, and SignalP-noTM, which is not annotated further but
+seems to indicate a signal peptide. The three ORFs lie on the positive strand, so we will name them in downstream order:
+ORF1 (522-9416 nt), ORF2 (9417-9509 nt), and ORF3 (9906-10388 nt). These ORFs are predicted to have the following
+domains:
+
+- An *Alphavirus*-like methyltransferase (MTase) domain (IPR002588) in ORF1.
+- An rRNA MTase FtsJ domain (IPR002877) in ORF1.
+- An alpha-ketoglutarate-dependent dioxygenase AlkB-like domain (IPR027450) in ORF1.
+- A RNA virus helicase core domain (IPR027351) in ORF1.
+- An RdRp from *Alsuviricetes* (IPR001788) in ORF1.
+- A transmembrane helix (TMh) that is part of a membrane-bound protein in ORF2.
+- A Tobacco mosaic virus-like coat protein (TMVC) (IPR001337) in ORF3.
+
+We are unable to find any elements encoding for non-coding RNAs in `Hriv_contig`, using Infernal cmscan 1.1.2. The
+following figure summarizes the contig-wide coverage, ORFs, predicted domains, and RdRp A, B, C motifs.
+
+![](imgs/hespivirus_rivalense/contig.png)
```

Comment:
Add a Figure Title, and a proper Figure Legend to your figures here. Below the image it should say *Figure X* and give a brief description. Please do add a short description of the methods you used for this figure. I really like it and would like to use this one as an example in the future, so document `bowtie2` if that's what you used etc..